kitchenci/.kitchen.yml

```diff
@@ -0,0 +1,16 @@
+---
+driver:
+  name: docker
+
+provisioner:
+  name: puppet_apply
+  puppet_version: "3.8.1"
+  mainfests_path: manifests
+  modules_path: modules
+
+platforms:
+  {platform-placeholder}
```

Comment:
actually in jenkins job itself there is a step that substitute the {platform-placeholder} with value of ${Distribution} parameter as following 

sed -i -e "s/{platform-placeholder}/${Distribution}/" .kitchen.yml